### ISSUE TITLE
Tests

### ISSUE THREAD

**Issue by amilajack** [aredridel/node-bin-gen#50]
I'd love to write some tests for this project. Would a PR for that be welcome?

**Comment by aredridel** [comment]
Absolutely! A fair bit of reorganization may be necessary to make it easily testable, but I'd be happy to see that!
